Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/orchestration/AbstractOrchestrable.java

```diff
@@ -44,9 +44,8 @@ protected AbstractOrchestrable() {
 
     }
 
-    protected AbstractOrchestrable(PolicyTransformer pt, AuditTransformer at) {
+    protected AbstractOrchestrable(PolicyTransformer pt) {
         policyTransformer = pt;
-        auditTransformer = at;
     }
 
     public AuditTransformer getAuditTransformer() {
```

Comment:
Do we still use the AuditTransformer for anything?  I can't find where the getter or setter is used.